app/models/target.rb

```diff
@@ -44,29 +43,29 @@ class Target < ApplicationRecord
   scope :team, -> { where(role: ROLE_TEAM) }
   scope :sessions, -> { where.not(session_at: nil) }
 
-  ROLE_STUDENT = 'student'
-  ROLE_TEAM = 'team'
+  ROLE_STUDENT = "student"
```

Comment:
This file has no other change other than re-format; can you be careful not to push files with no changes related to the issue?